docs/pages/components/form.md

```diff
@@ -94,16 +94,14 @@ Do not use the input field if:
     <div class="fd-form-item fd-form-item--horizontal fd-has-margin-none fd-popover">
         <label class="fd-form-label" for="input-1">Default input:</label>
         <div class="fd-form-input-message-group">
-            <input class="fd-input fd-popover__control" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label" aria-controls="popoverD1" aria-expanded="false" aria-haspopup="true">
-            <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message" aria-hidden="true" id="popoverD1">Normal message</span>
```

Comment:
If you are removing the form message it should not be wrapped in a fd-form-message-group